requirements.txt

```diff
@@ -4,6 +4,7 @@ urllib3>=1.24.2,<=1.26.8                                    # HTTP library with
 dogpile.cache>=1.1.2,<=1.1.5                                # Caching API plugins (1.1.2 is the first version to support pymemcache)
 tabulate~=0.8.0                                             # Pretty-print tabular data
 jsonschema~=3.2.0                                           # For JSON schema validation (Policy modules)
+dataclasses==0.8.0; python_version == '3.6'                  # For dataclass support in 3.6
```

Comment:
Comment alignment, plus the comment itself is useless. I am not in favor of documenting it because everyhting else is when it adds not value.